app/assets/stylesheets/application.tailwind.css

```diff
@@ -4,3 +4,17 @@
 @import "tailwindcss/overrides";
 
 @import "components/pagy";
+
+button .show-when-disabled {
```

Comment:
I think we might be able to handle this just with Tailwind psuedo-classes. For instance `disabled:show`. In fact, we might be able to eliminate the conditional from the component too. [Sean gives a nice example in his PR for disabling in Turbo.](https://github.com/hotwired/turbo/pull/386#issue-993156012)

Reply:
+1 on giving Tailwind pseudo-classes a go, if possible!